docs/create.md

```diff
@@ -0,0 +1,33 @@
+# Creating a new event
+
+To create and publish an event to the activity app, a new `IEvent` should be fetched from the activity manager and afterwards be passed to the `publish()` method:
+
+```php
+$event = \OC::$server->getActivityManager()->createEvent();
```

Comment:
we should also mention that IManager can be injected into classes, which is obviously preferred over querying the server directly. Expect people to copy sample code from the docs :-)